protocol/communities/manager_archive.go

```diff
@@ -234,7 +234,10 @@ func (m *ArchiveManager) GetCommunityChatsFilters(communityID types.HexBytes) (m
 
 	filters := messagingtypes.ChatFilters{}
 	for _, cid := range chatIDs {
-		filters = append(filters, m.messaging.ChatFilterByChatID(cid))
+		filter := m.messaging.ChatFilterByChatID(cid)
+		if filter != nil {
```

Comment:
I am wondering whether this is caused by `messaging` facade or it was there before as well. @jrainville would you be able to check?